frontend/package.json

```diff
@@ -13,9 +13,20 @@
         "lint": "npm run lint:type && npm run lint:js",
         "start:dev": "vite",
         "build": "tsc -p tsconfig.build.json && vite build",
-        "preview": "vite preview"
+        "preview": "vite preview",
+        "storybook": "storybook dev -p 6006",
+        "build-storybook": "storybook build"
     },
     "devDependencies": {
+        "@storybook/addon-essentials": "7.6.17",
+        "@storybook/addon-interactions": "7.6.17",
+        "@storybook/addon-links": "7.6.17",
+        "@storybook/addon-onboarding": "1.0.11",
+        "@storybook/blocks": "7.6.17",
+        "@storybook/builder-vite": "7.6.17",
+        "@storybook/react": "7.6.17",
+        "@storybook/react-vite": "7.6.17",
+        "@storybook/test": "7.6.17",
```

Comment:
are you sure that we need all these packages - https://storybook.js.org/docs/builders/vite

Reply:
They were installed automatically with the installation command, even prop types, I suppose they were needed to run storybook. Well I uninstalled prop types and @storybook/test and still working fine. Are you sure you installed dependencies? because in my local works fine, the other dependencies in fact are used by main file.